utils/nx-plugin/index.js

```diff
@@ -0,0 +1,13 @@
+const {
+  addReleaseBuildTargets,
+} = require('./clone-build-targets-for-release.cjs');
+
+module.exports = /** @type {import('@nrwl/devkit').NxPlugin} */ ({
+  name: '@coveo/nx-plugin',
+  processProjectGraph(currentGraph) {
```

Comment:
This allows us to modify Nx's project graph completely. In our case, we use that to add new targets (tasks).

Alternatively, there's another function that allows us to add targets directly, but it doesn't allow us to read other targets, which we need to do.